pkgs/applications/networking/cluster/k3s/update.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl gnugrep gnused jq
+
+set -eou pipefail
+
+cd $(dirname "${BASH_SOURCE[0]}")
+
+LATEST_TAG_RAWFILE=$(mktemp)
+curl --silent ${GITHUB_TOKEN:+"-u \":$GITHUB_TOKEN\""} \
+    https://api.github.com/repos/k3s-io/k3s/releases/latest > ${LATEST_TAG_RAWFILE}
+
+LATEST_TAG_NAME=$(cat ${LATEST_TAG_RAWFILE} | jq -r '.tag_name')
```

Comment:
totally ignorable, but the cat can be removed via using `jq -r '.tag_name' ${LATEST_TAG_RAWFILE}` instead.
This applies to a few other `jq` invocations too.

Reply:
Done